python/sglang/srt/reasoning_parser.py

```diff
@@ -0,0 +1,150 @@
+import re
+from typing import Dict, Tuple
+
+
```

Comment:
There's one extra line here because I wasn't able to select multiple lines for suggestion :)
```suggestion
```